### DISCUSSION TITLE
Korean language + search option

### DISCUSSION THREAD

**Discussion by 1max3cricle** [openzim/zim-requests#692] [Ideas]
Hey i found your medical apps great, but Korean language is naccesery and so needed.

- And i asked about search in the page and didn't find.. for ex: opening a topic and I'm looking for some keywords in it and the search button search in the whole app not the opened page.

**Comment by Popolechien** [comment]
@1max3cricle Thanks. Generating an app depends on the number of available articles on the source wiki, We will look into it.

For your search issue you would need to open a ticket at https://github.com/kiwix/kiwix-android/issues (assuming you are on android)

Please indicate clearly which version of the app you are using, as well as any info that could help us reproduce the bug (e..g "I tried to look for this word, but instead got this result"; screenshots are also welcome)
